master/pom.xml

```diff
@@ -728,29 +727,12 @@
         <version>3.3.0</version>
         <scope>provided</scope>
       </dependency>
-      <dependency>
-        <groupId>commons-collections</groupId>
-        <artifactId>commons-collections</artifactId>
-        <version>3.2.2</version>
-        <scope>provided</scope>
-      </dependency>
```

Comment:
see SAK-49906, this is transitively needed

Reply:
needed by who?

Reply:
i think i understand.... you'd rather just declare it centrally instead of having old tool dependencies like velocity and tomahawk bring it in transitively